### ISSUE TITLE
Can't Craft realistic torches

### ISSUE THREAD

**Issue by HyperCodec** [MattCzyr/RealisticTorches#44]
The realistic torches won't craft, and only the normal torch recipe works. Maybe you could simply modify the vanilla torch instead of making a new item?

**Comment by voxelli** [comment]
I believe this has to do with `realistictorches:unlit_torch` not being properly registered. I don't have my development env set up, but it looks like the problem is a simple typo: https://github.com/MattCzyr/RealisticTorches/blob/9e1e7046b48095f6673c3506962d6c928624e7f0/src/main/java/com/chaosthedude/realistictorches/items/RealisticTorchesItems.java#L36

There is an extra `)`.

**Comment by Felix-FTW** [comment]
The mod was working fine until I made a copy of my world, now I can't craft unlit torches

**Comment by HyperCodec** [comment]
> The mod was working fine until I made a copy of my world, now I can't craft unlit torches

Do you know if this is fixed yet?

**Comment by HyperCodec** [comment]
This issue has been open for a year

**Comment by Felix-FTW** [comment]
yeah im pretty sure the mod is abandoned, it hasnt been updated past 1.16

i can probably get a guy to update it in the future
if you have the money, look up arcane studios on discord, they have mod
devs on there

in the meantime, look at the code i put in the comments or check out that
"Restric Recipes" mod

**Comment by HyperCodec** [comment]
> yeah im pretty sure the mod is abandoned, it hasnt been updated past 1.16 i can probably get a guy to update it in the future if you have the money, look up arcane studios on discord, they have mod devs on there in the meantime, look at the code i put in the comments or check out that "Restric Recipes" mod
> […](#)

I could probably update the mod myself — it can't be *that* hard — rather than making someone else do it. The problem I see here is, if this is truly abandoned, no one will be there to accept a pull request.

**Comment by MattCzyr** [comment]
> I could probably update the mod myself — it can't be _that_ hard — rather than making someone else do it. The problem I see here is, if this is truly abandoned, no one will be there to accept a pull request.

I'm still around and able to accept the PR if someone wants to update this mod. I just haven't had the time to do it myself as I've been prioritizing my other mods for quite a while.

**Comment by Felix-FTW** [comment]
You should put what you just said in the description of the mod so
people know what's up

**Comment by HyperCodec** [comment]
> I believe this has to do with `realistictorches:unlit_torch` not being properly registered. I don't have my development env set up, but it looks like the problem is a simple typo:
> 
> https://github.com/MattCzyr/RealisticTorches/blob/9e1e7046b48095f6673c3506962d6c928624e7f0/src/main/java/com/chaosthedude/realistictorches/items/RealisticTorchesItems.java#L36
> 
> There is an extra `)`.

I don't see how there is an extra `)`. Looks the same as the rest.

**Comment by MattCzyr** [comment]
I've got some time to come back to this mod, and I looked into this issue. I wasn't able to recreate it myself (in my testing, unlit torches can be crafted as intended), but I have an idea of what might be causing it for others. If either of you are still facing this issue, try this file and see if it persists:

https://www.dropbox.com/s/6lkoymjstb5hpu5/RealisticTorches-1.16.5-3.1.3-beta.jar?dl=0

Note, I also updated to the latest Forge version for 1.16.5, so make sure you're using the latest version.

**Comment by HyperCodec** [comment]
> I've got some time to come back to this mod, and I looked into this issue. I wasn't able to recreate it myself (in my testing, unlit torches can be crafted as intended), but I have an idea of what might be causing it for others. If either of you are still facing this issue, try this file and see if it persists:
> 
> 
> 
> https://www.dropbox.com/s/6lkoymjstb5hpu5/RealisticTorches-1.16.5-3.1.3-beta.jar?dl=0
> 
> 
> 
> Note, I also updated to the latest Forge version for 1.16.5, so make sure you're using the latest version.

I'll go find out what version and such I am using and get back to you (should be on Monday bc busy tomorrow)
